src/components/confirm/confirm.ts

```diff
@@ -12,15 +27,15 @@ function useNativeConfirm() {
         && window.confirm;
 }
 
-async function nativeConfirm(options) {
+async function nativeConfirm(options: string | ConfirmOptions) {
     if (typeof options === 'string') {
         options = {
             title: '',
             text: options
-        };
+        } as ConfirmOptions;
     }
 
-    const text = (options.text || '').replaceAll('<br/>', '\n');
+    const text = (options.text || '').replace('<br/>', '\n');
```

Comment:
This doesn’t seem like it should be changed?

Reply:
@StableCrimson could you take a look at this question when you have a minute? Thanks!

Reply:
Sorry for the late response. Is there ever a case in which we will be replacing more than one occurrence of <br />? I can undo if necessary

Reply:
I believe there could be.

Reply:
Updated!